kalite/main/models.py

```diff
@@ -279,7 +279,10 @@ def add_log_to_summary(cls, user_log, device=None):
             start_datetime__lte=user_log.end_datetime,
             end_datetime__gte=user_log.end_datetime,
         )
-        assert log_summary.count() <= 1, "There should never be multiple summaries in the same time period/device/user/type combo"
+
+        if log_summary.count() > 1:
+            for log in log_summary[1:]:
+                log.soft_delete()
```

Comment:
This can work as a first-pass; it would be good to add a TODO here indicating that we should try to better resolve this occurrence when it comes up, in the future, and try to get the bottom of why it's happening.